spec.html

```diff
@@ -1118,10 +1118,9 @@ <h1>Identity</h1>
   </emu-clause>
 </emu-clause>
 
-<emu-clause id="sec-ecmascript-data-types-and-values" aoid="Type">
+<emu-clause id="sec-ecmascript-data-types-and-values">
```

Comment:
We should have an `oldid` somewhere, maybe on SameType, as a hint to readers for the new style.

Reply:
Actually this doesn't create its own anchor, the call links directly to `#sec-ecmascript-data-types-and-values`, so we should be good.